src/test/java/com/luxoft/svinenkov/movieland/controller/MovieControllerTest.java

```diff
@@ -0,0 +1,98 @@
+package com.luxoft.svinenkov.movieland.controller;
+
+import com.luxoft.svinenkov.movieland.entity.Movie;
+import com.luxoft.svinenkov.movieland.service.impl.MovieServiceImpl;
+import com.luxoft.svinenkov.movieland.utils.JsonManualConverter;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@ContextConfiguration(locations = "classpath:/spring-test-config.xml")
+@WebAppConfiguration
```

Comment:
no need in 2 above - you are mocking all

Reply:
removed